test/e2e/cypress/support/commands.js

```diff
@@ -92,6 +92,18 @@ Cypress.Commands.add('longRecoveryLifespan', ({} = {}) => {
     return config
   })
 })
+Cypress.Commands.add('dontLoginUserAfterRegistration', ({} = {}) => {
+  updateConfigFile((config) => {
+    delete config.selfservice.flows.registration.after['password']
+    return config
+  })
+})
+Cypress.Commands.add('enableLoginForVerifiedAddressOnly', ({} = {}) => {
+  updateConfigFile((config) => {
+    config.selfservice.flows.login['after'] = {password: {hooks: [{hook: 'require_verified_address'}]}}
```

Comment:
Instead of a replacement this should probably be adding the hook to the array, not replacing it :)